SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebuggerEditor.cpp

```diff
@@ -0,0 +1,106 @@
+#include "Utils/SpatialDebuggerEditor.h"
+
+#include "LoadBalancing/GridBasedLBStrategy.h"
+#include "LoadBalancing/LayeredLBStrategy.h"
+#include "Interop/Connection/SpatialWorkerConnection.h"
+#include "SpatialCommonTypes.h"
+#include "Utils/InspectionColors.h"
+#include "Debug/DebugDrawService.h"
+#include "Editor.h"
+#include "LevelEditor.h"
+#include "Modules/ModuleManager.h"
+#include "Engine/World.h"
+#include "GameFramework/WorldSettings.h"
+#include "EngineClasses/SpatialWorldSettings.h"
```

Comment:
we prefer to split out gdk includes in a group before engine includes:
```suggestion
#include "EngineClasses/SpatialWorldSettings.h"
#include "Interop/Connection/SpatialWorkerConnection.h"
#include "LoadBalancing/GridBasedLBStrategy.h"
#include "LoadBalancing/LayeredLBStrategy.h"
#include "SpatialCommonTypes.h"
#include "Utils/InspectionColors.h"

#include "Debug/DebugDrawService.h"
#include "Editor.h"
#include "Engine/World.h"
#include "GameFramework/WorldSettings.h"
#include "LevelEditor.h"
#include "Modules/ModuleManager.h"
```